src/intgtests/test_pam_hbac.py

```diff
@@ -375,6 +380,8 @@ def _config_setup(self, host):
         confd['BIND_PW'] = self.driver.bind_pw
         confd['SSL_PATH'] = self.driver.ca_cert
         confd['HOST_NAME'] = host
+        if start_ssl == False:
```

Comment:
On Mon, May 23, 2016 at 05:55:04AM -0700, Pavel Reichl wrote:
> > @@ -375,6 +380,8 @@ def _config_setup(self, host):
> >          confd['BIND_PW'] = self.driver.bind_pw
> >          confd['SSL_PATH'] = self.driver.ca_cert
> >          confd['HOST_NAME'] = host
> > +        if start_ssl == False:
> 
> PEP8 complains: src/intgtests/test_pam_hbac.py:383:22: E712 comparison to False should be 'if cond is False:' or 'if not cond:'
> 
> But I don't see what's wrong with your version, so I don't really care.

It's just a style issue, Python programmers prefer "is False" over "==
False". I fixed that..